index.html

```diff
@@ -52,7 +52,6 @@ <h3>Contact Us</h3>
             copyright
         </footer>
```

Comment:
- [ ] [optional] It would be nice if the footer position was not fixed so that when you add new books the design doesn't get broken